src/cls/IPM/Main.cls

```diff
@@ -630,6 +635,21 @@ generate /my/path -export 00000,PacketName2,IgnorePacket2^00000,PacketName3,Igno
 		</example>
 	</command>
 
+<command name="unmap">
+	<summary>
+		Unmap %IPM package and rountines in specified namespaces. Will Skip non-mapped namespaces.
+	</summary>
+	<modifier name="namespaces" aliases="ns" value="true" description="Comma-separated namespaces in which IPM mapping needs to be deleted."/>
+	<modifier name="globally" aliases="g" description="Will unmap IPM in all explicit namespaces that currently do not have IPM installed. By default, this modifier is not set and will not install globally."/>
```

Comment:
Nitpick: "By default, this modifier is not set and will not install globally" is not appropriate here